docs/UserGuide.md

```diff
@@ -390,82 +416,84 @@ Examples:
 
-### Deleting a person : `delete`
+### Deleting a contact : `delete`
 
-Deletes the specified person from the address book.
+Deletes the specified contact from the address book.
 
 Format: `delete INDEX`
 
-* Deletes the person at the specified `INDEX`.
+* Deletes the contact at the specified `INDEX`.
 
-* The index refers to the index number shown in the displayed person list.
+* The index refers to the index number shown in the displayed contact list.
 
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
 
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
+* `list` followed by `delete 2` deletes the 2nd contact in the address book.
 
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `find Betsy` followed by `delete 1` deletes the 1st contact in the results of the `find` command.
 
 <br>
 
 --------------------------------------------------------------------------------------------------------------------
 
-### Adding a public address to a person : `addpa`
+### Adding a public address to a contact : `addpa`
 
-Adds a public address to a person.
+Adds a public address to a contact.
 
 Format: `addpa INDEX c/NETWORK l/WALLET_NAME pa/PUBLIC_ADDRESS`
 
-* Adds a public address to a person at the specified `INDEX`. The index refers to the index number shown in the
-  displayed person list.
+* Adds a public address to a contact at the specified `INDEX`. The index refers to the index number shown in the
+  displayed contact list.
   The index **must be a positive integer** 1, 2, 3, …​
 
 * The `NETWORK` parameter specifies the ticker name for each network (e.g., `BTC`, `ETH`,
   `SOL`, etc.). This field is **case-insensitive** and will be converted to UPPERCASE. <br />
   Allowed values: `BTC|ETH|SOL`.
 
-* The `WALLET_NAME` parameter specifies the wallet name to which the public address belongs. 
+* The `WALLET_NAME` parameter specifies the wallet name to which the public address belongs.
   This field is **case-insensitive** and will be stored as is.
 
 * The `PUBLIC_ADDRESS` parameter specifies the public address to be added.
-  The length of BTC/ETH/SOL public addresses should be more than 26 characters, less than 44 characters and can only 
+  The length of BTC/ETH/SOL public addresses should be more than 26 characters, less than 44 characters and can only
   contain alphanumeric characters.
   This field is **case-insensitive** and will be converted to lowercase.
 
 Examples:
 
-* `addpa 1 c/ETH l/wallet1 pa/0x28f91d6e72eaf4372892e6c6e45dc41b574163e9fcdf94f4997958b46d772fa2` adds a public address
-  to the first person on the list `Alex Yeoh` under the ETH network with the wallet name `wallet1` and the public
-  address `0x28f91d6e72eaf4372892e6c6e45dc41b574163e9fcdf94f4997958b46d772fa2`.
+* `addpa 1 c/ETH l/wallet1 pa/0x0b1c9e1fb5e13c797c7f0134641810e9a7ca14d2
+* ` adds a public address
```

Comment:
Maybe adds a public address should be connected with the command, instead of being a new point?

Reply:
good catch